example/get-playlist-items.js

```diff
@@ -4,7 +4,7 @@ var config = require('./config');
 var youTube = new YouTube();
 
 youTube.setKey(config.key);
-youTube.getPlayListsItemsById('PLpOqH6AE0tNhInmRTSNf9f6OQsdaSJS8F', function(error, result) {
+youTube.getPlayListsItemsById('PLpOqH6AE0tNhInmRTSNf9f6OQsdaSJS8F', 8, function(error, result) {
```

Comment:
Hmm just wondering, `getPlayListItemsById` now has 4 arguments - `(id, maxResults, pageToken, callback)` 

* Why is it called with only 3 arguments here?
* Why is the maxResults set to such a specific number as 8?

Reply:
changed it to config.max